cmd/authelia-scripts/cmd_suites.go

```diff
@@ -143,6 +144,22 @@ func runSuiteSetupTeardown(command string, suite string) error {
 	cmd.Stderr = os.Stderr
 	cmd.Env = os.Environ()
 
+	suiteEnv := "internal/suites/" + selectedSuite + "/.env"
```

Comment:
Great! authelia-suites binary already handles the suite resources, I think it would be better to move it there so that everything related to resources is colocated. It will avoid the duplication of code too. What do you think?